### DISCUSSION TITLE
Location where FX Open

### DISCUSSION THREAD

**Discussion by rncwalker** [apohl79/audiogridder#1049] [Ideas]
When I use the AG plugin, it opens a capture of the plugin only. I have some plugins that have a menu on the right side and when I use it, it opens to the right.
To fix this problem I need to enlarge the capture area to see the menu. The problem with this is that if you move is another part of the DAW and then back to AG, you need to enlarge the capture area again.

AG needs two things prevent this problem.

1) On the AG Server, allow user to set size (Area) of what's captured and allow FX to open on right side of screen on the server.
2) On the DAW AG plugin, allow user to adjust size of capture window and lock, so that it would not change.

Robert Walker

**Comment by rncwalker** [comment]
The only way I found to get this to work, is to Change my display on the Server from 256x1440 to 1024x768.
That at least works for this one particular FX, hopefully it will not cause problems with other FX's

Good thing the Server is only for AudioGridder!

Robert

**Comment by apohl79** [comment]
I think I don't understand your issue and why you can solve it when you change the desktop resolution.

If a plugin opens a menu, those extended dimensions can't easily be detected. Thats why you can manually extend the capture area. Once extended, the state is stored. So not sure what you mean by lock?

**Reply by rncwalker** [reply]
Here's my AG Server at 1900x1080

![1-AG_Server MaxDisplay-](https://user-images.githubusercontent.com/112222499/199132880-cd5d8bc5-4d9a-4b91-bfe7-fe2b5bde84d4.png)

 
Here is what it looks like in my DAW FX bin (Reaper)

![2-AG_Plugin MaxDisplay-](https://user-images.githubusercontent.com/112222499/199132904-d6e371fd-5b01-42e1-a46d-a4a477c3e724.png)



You are correct it that you can extend the area and the state is stored during the project use. If you save the project and later come back to the project, you will need to extended the area again, hence it was not saved with the project.

Now by changing the screen size on the AG Server to the size you need (MY care it's 102x768) you get this on the server

![3-AG_Server 1024x768Display-](https://user-images.githubusercontent.com/112222499/199132929-6afe0d7b-36af-41b3-8117-01ac5d2bf123.png)


and this on the DAW plugin

![4-AG_Plugin 1024x768Display-](https://user-images.githubusercontent.com/112222499/199132945-f01e4fc8-8751-4e31-8468-ed3b35ec94fe.png)


I found things work best by having the Capture turned off and use Windows RDC (Remote Desktop Connection), I've had no problems with it, just do not disconnect or close as you will loose connection.
just minimize the window if needed. to get the window back select the RDC icon on your task bar. doing it this way allows me to get a 3ms latency.


![5-Using RDC](https://user-images.githubusercontent.com/112222499/199132991-fc9fa986-262d-4a0e-91fb-20557b4b84bf.gif)

Robert

**Reply by apohl79** [reply]
Ok. The capture area could also be persisted in the DAW session to survive a session reload. Will add a feature request for this. Thx!

**Comment by rncwalker** [comment]
First, your program "AudioGridder" is fantastic!

you said that the state is "stored" , If that means that always going back to the settings you made, then that's what I mean by "Lock"

The problem is, it does not stay at the settings you make.

I will work on making a video showing you what is happening.

Thanks,

Robert :)

**Comment by rncwalker** [comment]
This video clip (No sound) is showing the **DAW with the AG server** running on the same system. I am on a multi monitor setup and here is the  problem with where I place the FX and how it goes back to the other monitor by default. when I change settings, also happened when I save the project and then later, when I again load the project, it in in it's default location.

![AGPlug1](https://user-images.githubusercontent.com/112222499/198858547-41b6b033-9583-474d-b6fd-568440eaeee3.gif)

Audio gridder needs to allow the user to set where different windows open at. "Show Statistics" window stay where the user places it, the other windows do not. "The Show Monitor..." does not even let you move it.

On the **AG Server**, the user should set what area is captured and not the entire screen (Mine is **2560x 1440**).
I have found that by not using the capture, I can get as low as 3ms. So maybe setting the capture area will lower the latency.
Also allow the user to place where the effects open on the screen.

Hope this gives you some understanding of what's happening.

Robert

**Reply by apohl79** [reply]
Do you have the "local mode" enabled in the server settings?

**Reply by rncwalker** [reply]
I did , but if you also have on the plugin set to Generic Editor, it does not show. Thanks for getting me to look farther into this.
Now working.

Robert

**Comment by rncwalker** [comment]
It might be good to also allow the FX to display on the server and have the capture off (To get the low latency), so that you could control the FX from the server and not the DAW!

Robert

**Reply by apohl79** [reply]
You can disable screen capturing in the server settings.

**Comment by rncwalker** [comment]
Is there a way to move the "AGridder Monitor" somewhere else on the screen?

**Reply by apohl79** [reply]
Nope. it is fixed to the primary display right now.
